packages/universal/debug/src/setup.ts

```diff
@@ -3,7 +3,9 @@ if (
   typeof require === "function"
 ) {
   try {
-    const buffer = await import("buffer");
+    // this is for CJS only, so require is the only option here
```

Comment:
I made this change because this is also needed in debug builds in the browser (otherwise the stack trace infra doesn't work). `await import` is the most general way of describing what we're doing, and it should be transpilable to `require` in CJS builds, no?

Reply:
We could do an iife, would that work?